# Type: test
# Subtype: T1
# English Summary: Remove unused import statements
# Chinese Summary: 删除未使用的导入语句
# Entry Number: 23

# === Git Diff Content ===

diff --git a/subprojects/gradle-core/src/test/groovy/org/gradle/process/internal/child/ImplementationClassLoaderWorkerTest.java b/subprojects/gradle-core/src/test/groovy/org/gradle/process/internal/child/ImplementationClassLoaderWorkerTest.java
index a3e8a43..80a6de1 100644
--- a/subprojects/gradle-core/src/test/groovy/org/gradle/process/internal/child/ImplementationClassLoaderWorkerTest.java
+++ b/subprojects/gradle-core/src/test/groovy/org/gradle/process/internal/child/ImplementationClassLoaderWorkerTest.java
@@ -19,7 +19,6 @@ package org.gradle.process.internal.child;
 import org.gradle.api.Action;
 import org.gradle.api.logging.LogLevel;
 import org.gradle.logging.LoggingManagerInternal;
-import org.gradle.process.child.SerializableMockHelper;
 import org.gradle.util.JUnit4GroovyMockery;
 import org.gradle.util.ObservableUrlClassLoader;
 import org.jmock.Expectations;
